hugegraph-test/src/main/java/com/baidu/hugegraph/core/TaskCoreTest.java

```diff
@@ -565,7 +575,38 @@ public void testGremlinJobAndRestore() throws Exception {
                .input(request.toJson())
                .job(new GremlinJob());
 
-        HugeTask<Object> task = builder.schedule();
-        return task;
+        return builder.schedule();
+    }
+
+    private static void sleepAWhile() {
+        sleepAWhile(SLEEP_TIME);
+    }
+
+    private static void sleepAWhile(long ms) {
+        try {
+            Thread.sleep(ms);
+        } catch (InterruptedException e) {
+            // ignore
+        }
+    }
+
+    public static class TestTaskCallable<V> extends TaskCallable<V> {
+
+        public TestTaskCallable() {
+            // pass
+        }
+
+        @Override
+        public V call() throws Exception {
+            Thread.sleep(SLEEP_TIME);
+            return null;
+        }
+
+        @Override
+        public void done() {
+            StandardTaskScheduler scheduler = Whitebox.getInternalState(
+                                              this.task(), "scheduler");
```

Comment:
why not call this.task().scheduler()

Reply:
HugeTask doesn't have scheduler()
Already change to this.graph().taskScheduler()